src/pages/index.js

```diff
@@ -37,7 +38,11 @@ function App() {
   useEffect(() => {
     async function initImjoy() {
       const { setupRPC } = await import('imjoy-rpc');
-      const api = await setupRPC({ name: 'viv-plugin' });
+      const api = await setupRPC({ 
+        name: 'vizarr',
+        description: 'A minimal, purely client-side program for viewing Zarr-based images with Viv & ImJoy',
+        version: vizarrVersion,
```

Comment:
As a side note, how do you version `kaibu`? When hosting the app on `gh-pages`, I'm not sure of the best way to ensure older versions are available.

Reply:
We use netlify for that, it can build different sites for releases and PRs ([example](https://github.com/imjoy-team/kaibu/pull/19#issuecomment-643771000)) as I mentioned in another PR. The setup is very easy, you basically connect the repo and fill in the a build command.

Reply:
BTW, could you test and see if you get the version work. For some reason, `npm run dev` didn't work for me.

Reply:
> For some reason, npm run dev didn't work for me.
What isn't working exactly?

Reply:
If I change the create window src to http://127.0.0.1:3000, it doesn’t work.

Reply:
It works now, just tried it. please ignore this, sorry.